codes.marzeq.xbox-cloud-gaming-electron.yaml

```
@@ -0,0 +1,56 @@
+app-id: codes.marzeq.xbox-cloud-gaming-electron
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+separate-locales: false
+command: start-xbox-cloud-gaming-electron
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create
+
+modules:
+  - name: xbox-cloud-gaming-electron
+    buildsystem: simple
+    build-commands:
+      - install -Dm 644 dest/codes.marzeq.xbox-cloud-gaming-electron.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm 644 dest/codes.marzeq.xbox-cloud-gaming-electron.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm 644 dest/codes.marzeq.xbox-cloud-gaming-electron.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.appdata.xml
```

```suggestion
      - install -Dm 644 codes.marzeq.xbox-cloud-gaming-electron.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
      - install -Dm 644 codes.marzeq.xbox-cloud-gaming-electron.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
      - install -Dm 644 codes.marzeq.xbox-cloud-gaming-electron.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.appdata.xml
```

Also you removed the actually copy of the content. That doesn't change.

---

Does the `dist` param for `type: archive` you mentioned actually change anything then?